### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by sergiozar** [linuxmint/nemo#2779]
```
 * Nemo version 5.03
 * Is issue with desktop or windowed nemo? same
 * Distribution - Mint 20.2
 * Graphics: Device-1: Advanced Micro Devices [AMD/ATI] Turks PRO [Radeon HD 6570/7570/8550] vendor: PC Partner Limited; driver: radeon v: kernel bus ID: 01:00.0 chip ID: 1002:6759 
 * 64 bit
 ```

**Issue**: When you open Nemo, after upgrading to LM Cinnamon 20.2, and you look at the side panel you can see the "Devices" NOW ordered in alphabetical order, so I must stop myself and THINK (because after many many years of mount order with the hand that goes at the right devices automatically) because I, WE all are, forced to read and search the right device. This is a time loss for many users and a security issue because I have encrypted disks, auto-mounted disks, fstab edited and mistake device can have consequences. 
I really ask to myself what mental dynamic moves the programmers to force the users to adapt and accept a changing in something that was already ok. Force all. I don't understand this.
What I can understand is to give the choice at the user to list the devices in mount order or alphabetical order, this I can understand...

**Steps to reproduce** open nemo and you'll see

**Expected behaviour** devices listed as choice of the user or in mount order as yesterday

**Other information**: I know, my manners suck, sorry for that, I'm in therapy from many years and it not goes very well either.
Anyway, thank You for read me, I hope you can give us a choice in matter

**Comment by Jeremy7701** [comment]
You'll soon get used to choosing the right device from all those devices you neglected to put in `fstab`.

**Comment by sergiozar** [comment]
Yes maybe, but you don't get to force the users to adapt to every "feature"
you dream by night because you eat too much at dinner.

Il giorno sab 10 lug 2021 alle ore 03:10 Jeremy7701 <
***@***.***> ha scritto:
